liveclasses/models.py

```
@@ -0,0 +1,64 @@
+                    .sign_with(reader.read())
+                return token
+        except Exception as e:
+            print(e)
+
+    def get_meeting_url_teacher(self):
+        """Return the secure Jitsi meeting URL with JWT authentication."""
+
+        token = self.generate_jwt_token()
+        return f"https://8x8.vc/{settings.JITSI_APP_ID}/{self.meeting_id}?jwt={token.decode()}"
+
+    def get_meeting_url_student(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_meeting_url_student`

Add return type annotation: `str`

(ANN201)